pyproject.toml

```diff
@@ -0,0 +1,114 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.version]
+path = "omc3/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+  "/omc3",
+]
+exclude = [
+  "/.github",
+  "/doc",
+  "/tests",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["omc3"]
+
+[project]
+name = "omc3"
+readme = "README.md"
+description = "An accelerator physics tools package for the OMC team at CERN."
+authors = [
+    {name = "OMC Team", email = "pylhc@github.com"},  # see zenodo file / commits for details
+]
+license = "MIT"
+dynamic = ["version"]
+requires-python = ">=3.9"
+
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering :: Physics",
+    "Topic :: Scientific/Engineering :: Visualization",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Typing :: Typed",
+]
+
+dependencies = [
+  "numpy >= 1.24",
+  "scipy >= 1.10",
+  "pandas >= 2.1",
+  "tfs-pandas >= 3.8",
+  "matplotlib >= 3.8",
+  "Pillow >= 6.2.2",  # not our dependency but older versions crash with mpl
+  "generic-parser >= 1.1",
+  "sdds >= 0.4",
+  "optics-functions >= 0.1",
+  "turn_by_turn >= 0.6",
+  "uncertainties >= 3.1",
+  "scikit-learn >= 1.0",
+  "h5py >= 2.9",
+  "tables >= 3.6",
+  "requests >= 2.27",
+]
+
+[project.optional-dependencies]
+cern = [
+  "jpype1 >= 1.3",
+  "pytimber >= 3.0",  # NXCALS support
+  "pylogbook >= 3.4",
+  "kerberos >= 1.3.1",  # requires having krb-5config installed on the system
+]
+optional = [
+  "pymupdf >= 1.22",  # logbook for conversion from pdf to png
+  "qtpy >= 2.3.1",  # putting plots in windows
+]
+test = [
+  "jpype1 >= 1.3",
+  "pytest >= 7.0",
+  "pytest-cov >= 2.9",
+  "pytest-timeout >= 1.4",
+  "gitpython >= 3.1",
```

Comment:
a quick search on github didn't show me that this was used anywhere in our code. Can you double-check?

Reply:
It imports as `git` in our conftest and you introduced it as a "better" way to get `acc-models-lhc` in fixtures.

Reply:
grrr... I even looked in `conftest` but didn't see it. And looked in model creation, as I assumed it had something to do with the acc-models... thanks for finding it!

Can you write `# imported as git` behind, for future reference? ^^